backend/user-service/db/connectDB.ts

```diff
@@ -0,0 +1,17 @@
+import mongoose, { Mongoose } from "mongoose";
+import { USER_SERVICE_MONGO_URI } from '../utils/config';
+
+const connectMongoDB = async () => {
+	try {
+		const conn: Mongoose = await mongoose.connect(USER_SERVICE_MONGO_URI);
+		console.log(`MongoDB connected: ${conn.connection.host}`);
```

Comment:
For some reason the indentation is off, but it is fine locally in my pc.

Reply:
in vscode `Ctrl+Shift+P` -> type in `Convert indentation to spaces`

Reply:
Ok fixed.